packages/iris-grid/src/IrisGridPartitionedTableModel.ts

```diff
@@ -0,0 +1,105 @@
+/* eslint class-methods-use-this: "off" */
+import type {
+  Column,
+  dh as DhType,
+  PartitionedTable,
+  Table,
+} from '@deephaven/jsapi-types';
+import { Formatter } from '@deephaven/jsapi-utils';
+import { ColumnName } from './CommonTypes';
+import IrisGridModel from './IrisGridModel';
+import IrisGridTableModel from './IrisGridTableModel';
+import EmptyIrisGridModel from './EmptyIrisGridModel';
+import { PartitionConfig, PartitionedGridModel } from './PartitionedGridModel';
+
+export function isIrisGridPartitionedTableModel(
+  model: IrisGridModel
+): model is IrisGridPartitionedTableModel {
+  return (
+    (model as IrisGridPartitionedTableModel).partitionedTable !== undefined
+  );
+}
```

Comment:
We shouldn't need this function, we already have `isPartitionedGridModel` method.

Reply:
This function is still used in `set partitionConfig` inside `IrisGridProxyModel` since how the next model is set depends on the which model type it is

Reply:
Shouldn't need to use that there